docs/pages/color_examples.js

```diff
@@ -0,0 +1,388 @@
+              text={
+                <Text>
+                  Colors should be applied purposefully as it can convey meaning in numerous ways.
+                  For example, light and dark mode themes have different color hues and saturation
+                  for better contrast; however, the colors should be consistent with{' '}
+                  <Link
+                    inline
+                    href="https://gestalt.pinterest.systems/color_palette#Extended-palette"
+                  >
+                    <Text inline underline>
+                      Pinterest&apos;s brand colors
+                    </Text>
+                  </Link>{' '}
+                  and purpose, following our color standards.
+                </Text>
+              }
+            />
+          </Flex.Item>
+          <Flex.Item flex="grow" flexBasis="0%" minWidth={260}>
+            <PrincipleLayout
+              color="blue-skycicle-500"
+              image={<A11Y />}
+              heading="Accessible information"
+              text={
+                <Text>
+                  All color pairings must pass WCAG&apos;s AA color contrast standards. Using our
+                  color tokens properly ensures our colors meet legibility standards. Check out our{' '}
+                  <Link inline href="https://gestalt.netlify.app/accessibility">
+                    <Text inline underline>
+                      accessibility guidelines
+                    </Text>
+                  </Link>{' '}
+                  for design guidance and color contrast tools information.
+                </Text>
+              }
+            />
+          </Flex.Item>
+        </Flex>
+      </MainSection>
+      <MainSection name="Color best practices">
+        <MainSection.Subsection columns={2}>
+          <MainSection.Card
+            cardSize="md"
+            type="do"
+            description="Use colors to support creating distinction between elements, such as define primary and secondary actions. See [color usage](/color_usage) for reference and appropriate tokens."
+            defaultCode={`
+            <Flex gap={4}>
+              <IconButton
+                icon="speech"
+                iconColor="darkGray"
+              />
+              <Button
+                color="gray"
+                text="Visit"
+              />
+              <Button
+                color="red"
+                text="Save"
+              />
+              <IconButton
+                icon="share"
+                iconColor="darkGray"
+              />
+            </Flex>
+`}
+          />
+          <MainSection.Card
+            cardSize="md"
+            type="don't"
+            description="Use color as a sole indicator of information. Color-only changes do not work well for those who may be color blind or have low vision; always supply an icon or text label for context."
+            defaultCode={`
+            <Flex direction="column" gap={2}>
```

Comment:
What would be the right version of this example
![Screen Shot 2022-04-14 at 1 04 50 PM](https://user-images.githubusercontent.com/10593890/163438495-53dc62d5-b229-4441-b51a-858c3bfb46d0.png)
?

Reply:
Something using symbols + color, like triangle, square etc.